samples/net/civetweb/common/src/libc_extensions.c

```diff
@@ -11,11 +11,23 @@ LOG_MODULE_REGISTER(lib_extensions, LOG_LEVEL_DBG);
 #include <string.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <errno.h>
 
 #include "libc_extensions.h"
 
 #define FN_MISSING() LOG_DBG("[IMPLEMENTATION MISSING : %s]\n", __func__)
 
+int fileno(FILE *stream)
```

Comment:
It is functional for as long as no filesystem is used. Could it be that `NO_FILESYSTEMS` is not defined?

Reply:
Yeah, I've got no filesystem. Didn't know about the NO_FILESYSTEM define. I can try that out.
But it does work, I am running it on my blackpill board.

Reply:
There is a civetweb Websocket sample written by me. I agree with the author, that it will work. I also agree, to merge this change, cause it stays in sample. But implementing some POSIX functions in Zephyr may be very difficult and maybe unnecessary due to the nature of the OS.